### ISSUE TITLE
spaCy integration ignores old entities

### ISSUE THREAD

**Issue by davidberenstein1957** [tomaarsen/SpanMarkerNER#17]
I think it might not be best practice to completely overwrite the previously obtained entities, maybe something like the code underneath would work better.

```python
from spacy.util import filter_spans

doc.set_ents(filter_spans(list(doc.ents) + new_ents))
```

**Comment by tomaarsen** [comment]
I'll look into adding this as a toggle, off by default. Thanks for the suggestion!

**Comment by tomaarsen** [comment]
Implemented in #16 and released in 1.3.0.
